shared/actions/teams.tsx

```diff
@@ -1378,82 +1378,63 @@ async function showTeamByName(action: TeamsGen.ShowTeamByNamePayload, logger: Sa
   ]
 }
 
-function* getMemberSubteamDetails(
+// See protocol/avdl/keybase1/teams.avdl:loadTeamTreeAsync for a description of this RPC.
+function* loadTeamTree(_: TypedState, action: TeamsGen.LoadTeamTreePayload, _logger: Saga.SagaLogger) {
+  yield RPCTypes.teamsLoadTeamTreeMembershipsAsyncRpcPromise(action.payload)
+}
+
+function* loadTreeTeamActivity(
```

Comment:
doesn't need to be a chainGenerator either. the rpc takes an array of keys, just make it and pass it to the rpc